packages/app-page-builder/src/templateEditor/config/BlockElementSidebarPlugin.tsx

```diff
@@ -61,7 +61,7 @@ const UnlinkTab = ({ permission }: UnlinkTabProps) => {
                     )}
                 </div>
                 <div className="info-wrapper">
```

Comment:
If you ask me, I'd just remove this. Or if we have an actual link, which at this point we may actually have, maybe we can link it finally.